tools/cldr-apps/js/src/views/AnnounceForm.vue

```diff
@@ -0,0 +1,152 @@
+<template>
```

Comment:
AnnounceForm and AnnouncePost are both child components of AnnouncePanel, the parent component